project.html

```diff
@@ -4,38 +4,38 @@
 permalink: project
 ---
 
-<p><strong>KeePassXC</strong> is a modern, secure, and open-source password manager that stores and manages your most sensitive information.</p>
+<p><strong>KeePassXC</strong> is a modern, secure, and open-source password manager that stores and manages your most sensitive info.</p>
 
-<p>You can run KeePassXC on Windows, macOS, and Linux systems. KeePassXC is for people with extremely high demands of secure
-personal data management. It saves many different types of information, such as usernames, passwords, URLs, attachments,
-and notes in an offline, encrypted file that can be stored in any location, including private and public cloud solutions.</p>
+<p>For Linux, macOS and Windows. Save many different types of info (such as usernames, passwords, URLs, attachments,
+and notes) in an offline, encrypted file that can be stored on your device, computer, server, or the cloud.</p>
 
-<p>For easy identification and management, user-defined titles and icons can be specified for entries. In addition, entries
-are sorted in customizable groups. An integrated search function allows you to use advanced patterns to easily find any entry
-in your database. A customizable, fast, and easy-to-use password generator utility allows you to create passwords with any
-combination of characters or easy to remember passphrases.</p>
+<p>Identify and manage entries with user-defined titles and icons, and customizable groups.
+Use advanced patterns in the integrated search function to easily find any entry in your database.
```

Comment:
Advanced queries.

Reply:
Maybe, but I think it complicates matters. It is less often that people know the word "query"(?)